### PR TITLE
Rework keyframes-remove-documentElement-crash.html to not use testharness.js

### LINKED ISSUE #20020: css/css-animations/keyframes-remove-documentElement-crash.html causes a harness failure
The test at [css/css-animations/keyframes-remove-documentElement-crash.html](http://web-platform-tests.live/css/css-animations/keyframes-remove-documentElement-crash.html) causes a harness failure because it removes the `<body>` element:

```
[Error] TypeError: null is not an object (evaluating 'root.appendChild')
	(anonymous function) (testharness.js:2987)
	(anonymous function) (testharness.js:3026)
	(anonymous function) (testharness.js:196)
	(anonymous function) (testharness.js:2776)
	forEach (testharness.js:3489)
	(anonymous function) (testharness.js:2773)
	all_complete (testharness.js:2643)
	(anonymous function) (testharness.js:2655)
	(anonymous function) (testharness.js:2446)
```

I'm not sure what the right way would be to address this given the test design. It does seem somewhat arbitrary that this test exists in the WPT suite though.

Tagging the author of the test @lilles, its reviewer @mstensho, other Blink developers working on Web Animations @stephenmcgruer @flackr and all-around knowledgeable person @birtles.

### PR DESCRIPTION
**Pull request by stephenmcgruer**
Previously this test would fail when loaded locally as the
testharness.js cleanup code would try to access
document.documentElement, which had been removed. Instead, use the
crashtest support
(https://web-platform-tests.org/writing-tests/crashtest.html).

Fixes #20020

### CODE REVIEW

**Comment by stephenmcgruer** [comment]
I'm not 100% sure this is working as I intend. By running with `--log-mach=- --log-mach-verbose --webdriver-arg=--verbose` and adding some logs to test-wait.js, I find that actually the code in test-wait.js isn't run until *after* the root has been removed. Which is fine (and I've verified that the rAF also runs first), I'm just sorta surprised about that...

File: css/css-animations/keyframes-remove-documentElement-crash.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <!doctype html>
+<html class="test-wait">
 <title>CSS Animations Test: Chrome crash when removing documentElement and @keyframes stylesheet</title>
 <link rel="help" href="https://crbug.com/999522">
 <script src="/resources/testharness.js"></script>
```
Comment (foolip) [inline css/css-animations/keyframes-remove-documentElement-crash.html]: Do you want to remove the script include too?

File: docs/writing-tests/crashtest.md [hunk 2]
```diff
@@ -12,11 +12,10 @@ The simplest crashtest is a single HTML file with any content. The
...
-In some cases crashtests may need to perform work after the initial
-page load. In this case the test may specify a `class=wait` attribute
-on the root element. The test will not complete until that attribute
-is removed from the root. At the time when the test would otherwise
-have ended a `TestRendered` event is emitted; test authors can use
-this event to perform modifications that are guaranteed not to be
-batched with the initial paint. This matches the behaviour of
-[reftests](reftests).
+In some cases crashtests may need to perform work after the initial page load.
+In this case the test may specify a `class=test-wait` attribute on the root
```
Comment (foolip) [inline docs/writing-tests/crashtest.md]: `class=test-wait` is the real change here, hard to spot because of line wrapping.

**Review by foolip — APPROVED** [review]
LGTM % nit

**Pull request merged**